patchwork-events-lifecycle/src/main/java/net/minecraftforge/fml/event/lifecycle/FMLFingerprintViolationEvent.java

```diff
@@ -31,10 +31,15 @@
  * <p>TODO: Patchwork will never fire this event. All jars that pass through patchwork are modified sweepingly.</p>
  */
 public class FMLFingerprintViolationEvent extends ModLifecycleEvent {
-	private final boolean isDirectory;
-	private final Set<String> fingerprints;
-	private final File source;
-	private final String expectedFingerprint;
+	private boolean isDirectory;
```

Comment:
instead of removing the final, instead set them to null explicitly in the eventbus constructor